src/handlers/dns/tests/test_dns.cpp

```diff
@@ -72,4 +72,31 @@ TEST_CASE("DNS Utilities", "[dns]")
         CHECK(result.first == ".b.c");
         CHECK(result.second == "");
     }
+
+    SECTION("aggregateDomain with static suffix")
+    {
+        AggDomainResult result;
+        std::string domain;
+
+        domain = "biz.foo.bar.com";
+        result = aggregateDomain(domain, ".bar.com");
```

Comment:
this is unintuitive i think - if we're passing a string, I would expect that string to match against domain in some way, so let's change it to a size_t